backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -239,11 +247,31 @@ public static Map<String, String> getNextRow(MappingIterator<Map<String, String>
   public static ValueOrError getValue(Map<String, String> row, String name, boolean isRequired) {
     String value = row.get(name);
     if (isRequired && (value == null || value.trim().isEmpty())) {
-      return new ValueOrError(new FeedbackMessage(ITEM_SCOPE, name + " is a required column."));
+      return new ValueOrError(new FeedbackMessage(ITEM_SCOPE, getRequiredValueErrorMessage(name)));
     }
     return new ValueOrError(value, name);
   }
 
+  public static List<FeedbackMessage> hasMissingRequiredHeaders(
+      Map<String, String> row, FileRow fileRow) {
+    List<FeedbackMessage> errors = new ArrayList<>();
+    Set<String> columns = row.keySet();
+    fileRow
+        .getRequiredFields()
+        .forEach(
+            requiredField -> {
+              if (!columns.contains(requiredField)) {
+                var feedback =
+                    new FeedbackMessage(
+                        CsvValidatorUtils.ITEM_SCOPE,
+                        "The header for column " + requiredField + " is missing or invalid.",
+                        List.of(1));
```

Comment:
⚠️ hardcoding 1 here because header will be first row -- let me know if it should be something else 😅

Reply:
I think it's also possible to construct a `FeedbackMessage` without a specific row number - that might be preferable in this case, since it's technically related to the first row but hardcoding that does feel a little brittle.

Reply:
Got it! Changed it to `null` and then will sort the error messages with the `null` indices to the top!